src/index.js

```diff
@@ -0,0 +1,2 @@
+import './api.js';
+import './style.css';
```

Comment:
- [ ] You have done great so far :+1: Fixing some changes requested by the previous reviewer. but there are some changes you still need to make before moving to the next milestone. :100: 
- [ ] Looking at the video below, you can see that I get an error when I start the server `Module not found: Error: Can't resolve 'css-loader' in ` the error seems to be pointing to a missing package, I'll suggest you fix this. :+1: :+1: 

- - Video reference:

https://github.com/paulshaun-kdtk/leaderboard/assets/107632780/bb2027bf-97c7-41ed-af78-74d849d31ed2

Reply:
hi @V-Blaze thanks for the review...

I'm  sorry but that error wasn't and still isn't reflecting on my local